packages/monosize-storage-azure/src/uploadReportToRemote.mts

```diff
@@ -24,6 +24,13 @@ export function createUploadReportToRemote(config: AzureStorageConfig) {
     commitSHA: string,
     localReport: BundleSizeReport,
   ): ReturnType<StorageAdapter['uploadReportToRemote']> {
+    // Validate the branch name
+    if (/[\\/]/.test(branch)) {
```

Comment:
q: is the escaping within `[]` necessary. AFAIR anything within the group is escaped automatically ?

or that's on purpose ? `double forward slash`